### PR TITLE
Cap requests version below 2.12.2

### PR DESCRIPTION
**Pull request by sbesson**
Last release seems to have broken docker-py /cc @joshmoore @aleksandra-tarkowska 

See https://github.com/ansible/ansible-modules-core/issues/5775 and
https://github.com/kennethreitz/requests/issues/3734 for the relevant issues

### CODE REVIEW

**Comment by joshmoore** [comment]
I'm still having issues with this:

```
RUNNING HANDLER [devspace : docker_service] ************************************
fatal: [adhoc-devspace]: FAILED! => {"changed": false, "failed": true, "msg": "Error: docker-py version is 1.10.6. Minimum version required is 1.7.0."}

```

File: ansible/roles/devspace/tasks/devspace-install.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@
     - passlib
     - pyyaml
     - docker-compose
+    # Temporary fix - cap requests to version below 2.12.2
+    # See https://github.com/kennethreitz/requests/issues/3734 and
+    # https://github.com/docker/docker-py/issues/1321
+    - requests<2.12.2
```
Comment (aleksandra-tarkowska) [inline ansible/roles/devspace/tasks/devspace-install.yml]: that is not needed here

Comment (aleksandra-tarkowska) [inline ansible/roles/devspace/tasks/devspace-install.yml]: my diff was:

```
$ git diff
diff --git a/ansible/roles/devspace/tasks/devspace-install.yml b/ansible/roles/devspace/tasks/devspace-install.yml
index b4bd39b..c6e69b4 100644
--- a/ansible/roles/devspace/tasks/devspace-install.yml
+++ b/ansible/roles/devspace/tasks/devspace-install.yml
@@ -59,6 +59,7 @@
     state: latest
   with_items:
     - docker-py==1.9.0
+    - requests<2.12.2
 
 - name: clone devspace
   become: yes
```

**Comment by manics** [comment]
You've added:
- `ansible/.clouds.cache`
- `ansible/plugins/callback/error_if_no_hosts_match.pyc`

**Comment by sbesson** [comment]
Thanks @manics force pushed.

**Comment by joshmoore** [comment]
Merging since this seems to work for @aleksandra-tarkowska. I'll keep trying.

File: ansible/roles/devspace/tasks/devspace-install.yml [hunk 2]
```diff
@@ -59,6 +59,8 @@
     state: latest
   with_items:
     - docker-py==1.9.0
+    # Same as above
```
Comment (aleksandra-tarkowska) [inline ansible/roles/devspace/tasks/devspace-install.yml]:
```
+    # Temporary fix - cap requests to version below 2.12.2
+    # See https://github.com/kennethreitz/requests/issues/3734 and
+    # https://github.com/docker/docker-py/issues/1321
```

could be added with the next PR

**Pull request merged**